packages/forklift-console-plugin/locales/en/plugin__forklift-console-plugin.json

```diff
@@ -110,7 +109,7 @@
   "Error: NFS mount end point should be in the form NFS_SERVER:EXPORTED_DIRECTORY, for example: 10.10.0.10:/ova.": "Error: NFS mount end point should be in the form NFS_SERVER:EXPORTED_DIRECTORY, for example: 10.10.0.10:/ova.",
   "Error: Password is required and must be valid.": "Error: Password is required and must be valid.",
   "Error: Please provide a valid URL with a schema, domain, and path. For example: https://rhv.com/ovirt-engine/api.": "Error: Please provide a valid URL with a schema, domain, and path. For example: https://rhv.com/ovirt-engine/api.",
-  "Error: Please provide a valid URL with a schema, domain, and path. For example: https://vcenter.com/sdk.": "Error: Please provide a valid URL with a schema, domain, and path. For example: https://vcenter.com/sdk.",
+  "Error: The provided URL is in an invalid format. Please provide a URL that includes a scheme, a domain name and path. For example: https://vCenter-host-example.com/sdk": "Error: The provided URL is in an invalid format. Please provide a URL that includes a scheme, a domain name and path. For example: https://vCenter-host-example.com/sdk",
```

Comment:
"a scheme, a domain name, and a path. For example, ..."

The comma instead of a colon is not absolutely necessary. If messages always us a colon after "For example," leave it alone.
The comma before "and" is necessary.